metis-dereference/metis-dereference-service/src/test/java/eu/europeana/metis/derefrence/test/MongoDereferenceServiceTest.java

```diff
@@ -21,6 +21,7 @@
 import eu.europeana.enrichment.api.external.EntityWrapper;
```

Comment:
Is there a reason for all the System.out.println statements? They don't really belong in test code.

Reply:
Agreed. I've removed all those pesky printlns.